src/pages/home/ethos/Ethos.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import harness from '@dojo/framework/testing/harness';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import { add } from '@dojo/framework/has/has';
+import * as css from './Ethos.m.css';
+import Ethos from './Ethos';
+import Card from '../../../widgets/card/Card';
+import FontAwesomeIcon from '../../../widgets/icon/FontAwesomeIcon';
+
+describe('Ethos', () => {
+	it('renders', () => {
+		add('build-time-render', true, false);
```

Comment:
Out of curiosity what does this do here? Do we need the build-time-render flag for this unit test since the `Ethos` widget never checks it?

Reply:
Good point, shall remove